includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -521,7 +519,7 @@ private function wc_setup_redirect_to_wc_admin_onboarding() {
 		$onboarding_data = get_option( 'wc_onboarding_profile', array() );
 		update_option( 'wc_onboarding_profile', array_merge( $onboarding_data, $profile_updates ) );
```

Comment:
@joshuatf `wc_onboarding_profile` should also be updated.

Reply:
Good catch, @rrennick